test/test_tools.py

```diff
@@ -49,7 +49,7 @@ def test_xmlupload(self):
                    imgdir="testdata/bitmaps",
                    sipi="http://0.0.0.0:1024",
                    verbose=True,
-                   validate=True)
+                   validate=False)
```

Comment:
@lrosenth @subotic To me it seems that the xml upload was not executed against DSP-API because `validate` was set to true. Hence, the upload was never tested automatically.